src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts

```diff
@@ -77,9 +99,20 @@ export class UmbDocumentTypeWorkspaceViewStructureElement extends UmbLitElement
 				</umb-property-layout>
 			</uui-box>
 			<uui-box headline="Presentation">
-				<umb-property-layout alias="Root" label="Collection view">
-					<div slot="description">Provides an overview of child content and hides it in the tree.</div>
-					<div slot="editor"><uui-toggle label="Display children in a Collection view"></uui-toggle></div>
+				<umb-property-layout alias="collection" label="Collection">
+					<div slot="description">
+						Configures the content item to show list of its children, the children will not be shown in the tree.
```

Comment:
Would be good to include the localization key for this

Reply:
@iOvergaard I'll update this PR with the localization keys that are related to this PR and do the separate PR for the rest of the Document Type Structure view.